clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBStoragePartition.java

```diff
@@ -375,6 +379,14 @@ public synchronized void endBatchWrite() {
     rocksDBSstFileWritter.ingestSSTFiles(rocksDB, columnFamilyHandleList);
   }
 
+  /**
+   *  After persisting EOP, delete the temp sst files that were copied to RocksDB
+   */
+  @Override
+  public void cleanupAfterBatchWrite() {
+    deleteSSTFiles(fullPathForTempSSTFileDir);
```

Comment:
Previously, `deleteSSTFiles` should happen rarely and now, it will happen regularly for every data push, so I am thinking whether we should enforce a throttler here to reduce the impact to local SSD, which may cause some latency spike because of one-time massive deletion.

Can we introduce some kind of rate limiter here? 
For example, expose some config to control a maximum number of files should be deleted per second, and we need to do some test in cert workflow to figure out a good number to reduce the impact to the read path.

Reply:
Couple of different problems here:
1. Files of multiple partition replicas being deleted at the same time: I am thinking of reusing the copy throttler for this as well. Only 1 copy/delete can happen at any time. What do you think?
2. When deletion of 1 partition replica's temp sst files happens, multiple files worth GBs will get deleted at the same time. Need to think about this.